parent/pom.xml

```diff
@@ -3046,6 +3048,11 @@
                     <artifactId>maven-plugin-plugin</artifactId>
                     <version>${maven-plugin-plugin-version}</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.eclipse.sisu</groupId>
+                    <artifactId>sisu-maven-plugin</artifactId>
+                    <version>${sisu-maven-plugin-version}</version>
+                </plugin>
```

Comment:
Why it need to introduce `sisu-maven-plugin`?

Reply:
As per Maven's documentation: https://maven.apache.org/maven-jsr330.html#how-to-use-jsr-330-in-plugins